src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Circle has been drawn");

```

Comment:
That's all the information you need about the figure?
<img width="808" alt="Screen Shot 2022-12-18 at 5 29 51 PM" src="https://user-images.githubusercontent.com/90692308/208306651-7018afac-beb9-4de5-9cd8-094744eb030e.png">